src/cxx.cc

```diff
@@ -33,6 +33,8 @@ void cxxbridge1$string$clone(rust::String *self,
                              const rust::String &other) noexcept;
 bool cxxbridge1$string$from(rust::String *self, const char *ptr,
```

Comment:
Please rename this symbol now that it's no longer the unique way to construct a string from data.

```suggestion
bool cxxbridge1$string$from_utf8(rust::String *self, const char *ptr,
```

Reply:
Done